src/py42/clients/devices.py

```diff
@@ -238,3 +238,30 @@ def get_settings(self, guid, keys=None):
         uri = u"/api/v4/device-setting/view"
         params = {u"guid": guid, u"keys": keys}
         return self._session.get(uri, params=params)
+
+    def get_agent_state(self, guid, property_name):
+        """Gets the agent state of the device.
+            `REST Documentation <https://console.us.code42.com/swagger/index.html?urls.primaryName=v14#/agent-state/AgentState_ViewByDeviceGuid>`__
+
+            Args:
+                guid (str): The globally unique identifier of the device.
+                propertyName (str): The name of the property to retrieve (e.g. `fullDiskAccess`).
```

Comment:
Casing in doc str for this param still camel

Reply:
I can approve after this!

Reply:
Should now be updated!